examples/with-ant-design/pages/index.tsx

```diff
@@ -26,6 +28,10 @@ export default function Home() {
   }
 
   return (
+<div>
+  <Head>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/antd/dist/antd.css" />
```

Comment:
https://nextjs.org/docs/messages/no-stylesheets-in-head-component#document

Reply:
thanks for the heads up ! I'll make the necessary changes.

Reply:
Hi @B2o5T 
I have made the changes according to your suggestion.